cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1650,7 +1650,7 @@ public int getTerminalWidth() {
     @Override
     public int getTerminalHeight() {
         if( !INTERACT ){
-            return 0;
+            return 40;
```

Comment:
Strangely, in the same method when an error occurs, 24 is returned. This value seems random. That is strange to have 24 returned in one case and 40 in another.  I looked at GIT history but nothing special. You should perhaps try to replace 24 by 40, I can't see how this could have an effect.